backend/presentation/views.py

```diff
@@ -0,0 +1,49 @@
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.viewsets import ModelViewSet
+
+from presentation.models import Presentation, Tag
+from presentation.permissions import IsAdminOrOwner, IsAdminOrReadOnly
+from presentation.serializers import (
+    InputPresentationSerializer,
+    OutputPresentationSerializer,
+    TagSerializer,
+)
+from presentation.services import PresentationService
+
+
+class PresentationViewSet(ModelViewSet):
+    queryset = Presentation.objects.all()
+    serializer_class = OutputPresentationSerializer
+    permission_classes = [IsAdminOrOwner]
+    services = PresentationService()
```

Comment:
In that case I would just call service directly in methods, since assigning it as class attribute doesn't grant us with any generic behavior